pkgs/applications/version-management/sublime-merge/common.nix

```diff
@@ -17,7 +17,7 @@ let
   archSha256 = sha256;
   arch = "x64";
 
-  libPath = lib.makeLibraryPath [ xorg.libX11 glib gtk3 cairo pango ];
+  libPath = lib.makeLibraryPath [ xorg.libX11 glib gtk3 cairo pango curl.out ];
```

Comment:
```suggestion
  libPath = lib.makeLibraryPath [ xorg.libX11 glib gtk3 cairo pango curl ];
```
This isn't working?

Reply:
@SuperSandro2000 Did you test it? I was not able to get using "curl" to work, only "curl.out". I'm running NixOS 21.05 and for me the "curl" package resolves to "/nix/store/8gdscs0q835k0mwrvyg5d7lbrkdakcmp-curl-7.79.1-bin" which contains only the curl binaries, whereas "curl.out" resolves to "/nix/store/3m3bh78qlizlywqkqnixqldghyy5lx38-curl-7.79.1" which contains the lib folder and other support files like the shell integrations. Please let me know if there's some better way of getting access to the libcurl libraries, I was not able to find one.

Reply:
curl.lib gets you only the libraries.

Reply:
@SuperSandro2000 I tried that, this is the error I get when I try to use curl.lib instead of curl.out. It appears the curl package does not provide the lib attribute.

error: while evaluating the attribute 'curl.lib' at /data/home/projects/nix/nixpkgs/pkgs/top-level/all-packages.nix:4363:3:
attribute 'lib' missing, at /data/home/projects/nix/test.nix:4:3

Reply:
🤔 curl's first output is bin. Oh well, whatever.

Reply:
Thanks @SuperSandro2000 !